translator/gobgp.py

```diff
@@ -22,14 +22,31 @@ def _get_family(self, ip_version):
         else:
             return gobgp_pb2.Family.AFI_IP
 
-    def _build_path(self, ip):
+    def _build_path(self, ip, event_data):
```

Comment:
I don't think it would necessarily hit us here, but we probably shouldn't set a mutable default (Ask me how I know 🥲). We should probably set it to `None` and then initialize it inside the function if we want it to be a default dict. 

I hate this "feature" of python.

https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html

Reply:
Something like this I think would be safe:

```python
def _build_path(self, ip, event_data=None):
        # Initialize event_data in case we didn't get any
        if not event_data:
            event_data = {}

        # Grab ASN and Community from our event_data, or use the defaults
        asn = event_data.get("asn", 65400)
        community = event_data.get("community", 666)
```